### ISSUE TITLE
pictures of PI

### ISSUE THREAD

**Issue by tpoisot** [bios2/bios2-website#7]
We should have picture of PI in the People page.

**Comment by aammd** [comment]
I agree! We could simply use the headshots from their websites, after emailing them to ask for permission. The table on the "People" page could be reworked as a list
